readme.txt

```diff
@@ -99,80 +99,37 @@ There are opportunities for developers at all levels to contribute. [Learn more
-* Release post: https://wp.me/p1moTy-foJ
-
-**Bug fixes**
-
-* General: avoid conflicting with other plugins when suggesting Jetpack features on the Plugins screen.
-* Publicize: avoid errors when the feature is not active on a site.
-* Widgets: improve performance of the Top Posts and the Blog Stats widgets on high-traffic sites.
-* Subscriptions: fix an issue that prevented displaying subscribers count in the subscription forms.
-* Tiled Galleries / Slideshows: ensure they can be displayed properly in Internet Explorer 11.
-
-= 7.1 =
-
-* Release date: March 05, 2019
-* Release post: https://wp.me/p1moTy-e9x
+* Release date: April 02, 2019
+* Release post: https://wp.me/p1moTy-foe
 
 **Major Enhancements**
 
-* Block Editor: this release introduces 6 new blocks:
-	- the Ads block allows you to insert different ads from [our WordAds program](https://jetpack.com/support/ads/) within your posts and pages.
-	- the Mailchimp block allows your readers to easily subscribe to your Mailchimp newsletter.
-	- the Video block supports VideoPress videos if you've purchased our Premium or Professional plan.
-	- the Slideshow block allows you to insert beautiful slideshows in your posts and pages.
-	- The Business Hours blocks is useful for companies who want to display their business's Opening Hours on their site.
-	- The Contact Info block is useful for any business who may want to display useful information on a post or page.
+* Adds a Repeat Visitor block that controls block visibility based on how often a visitor has viewed the page.
+* Selectively disable Ads blocks for mobile devices.
+
 
 **Enhancements**
 
-* Admin Page: move Carousel settings from Performance to Writing section.
-* Ads: include search results pages under the `Archive` toggle.
-* Block Editor: improve block registration structure for better management of block availability.
-* General: remove IE8 support fallbacks.
-* General: add feature suggestions to the plugin search screen.
-* Image CDN: add new mode that disables the creation of resized images, thus saving disk space.
-* Instagram: update embed to support Instagram TV URLs.
-* Post Images detection: add support for alt text.
-* Plans: clarify upgrade prompts in the Jetpack dashboard's Plans page.
-* Publicize: remove the Google+ interface as the Social Network is now deprecated.
-* Related Posts: update block to allow for up to 6 related posts.
-* Social Menus & Icons: add Discord Support.
-* Support: add additional tests to check when Jetpack isn't working as expected and ensures all current debugging platforms use the same testing list.
-* Simple Payments: add generic currency fallback symbol.
-* Sync: further performance improvements in PHP 7+ environments.
-* Woocommerce Analytics: include product type with analytics data.
+* Admin Dashboard: Various text and design improvements to make your administration experience all the better, including:
+** My Plan page is now more useful and actionable for free plans.
```

Comment:
You have an extra `*` in here.

Reply:
I was thinking of having it be a nested list if there was enough Admin Dashboard related things to note. There my not be and would condense that down.

Reply:
Oh I see, that makes sense now. I would condense it into one changelog entry, to be honest. I don't think the details are going to be very interesting for most people.